src/test/e2e/api/admin/state.e2e.test.ts

```diff
@@ -452,15 +452,15 @@ test(`should not show environment on feature toggle, when environment is disable
         .attach('file', 'src/test/examples/import-state.json')
         .expect(202);
 
-    await app.request
-        .post('/api/admin/projects/default/environments')
-        .send({ environment: 'state-visible-environment' })
-        .expect(200);
```

Comment:
This method was connecting the environments in the test, but when customers run the import and not this method they may run into this issue